Added Calibration command to OBC simulator

IHamed98 opened the pull request.

IHamed98 committed:
Added citiroc calibration command

tstana commented on obcsim/RS232.cpp:
```diff
@@ -127,6 +127,15 @@ void RS_read(msp_link_t *lnk)
       break;
 
+    case CMD_SEND_CITI_CALIB_CONF:
+        DEBUG_PRINT("CMD_SEND_CITI_CALIB_CONF received");
+        len = 4;
+        if (fill_commanddata(len)) {
+            DEBUG_PRINT("Invoking MSP_OP_SEND_CUBES_CITI_CALIB_CONF");
+            invoke_send(lnk, MSP_OP_SEND_CUBES_CITI_CALIB_CONF, commanddata, len, BYTES);
+        }
+        break;
```
Please use two spaces for a TAB, so we keep to the (bad, I agree) style of the code in this file.

tstana reviewed (changes requested):
I was hoping I'd finally be able to do a merge without a single comment, but something _always_ has to creep through, doesn't it?

Anyway, really simple change, should be straightforward to fix.

IHamed98 committed:
Fixed comments and  renamed commands

The indentation has been fixed, in addition the commands has been renamed for clarity.
